ambry-store/src/main/java/com.github.ambry.store/IndexSegment.java

```diff
@@ -244,11 +249,11 @@ File getFile() {
   }
 
   /**
-   * The key size in this segment
-   * @return The key size in this segment
+   * The persisted entry size of this segment
+   * @return The persisted entry size of this segment
    */
-  int getKeySize() {
-    return keySize;
+  int getPersistedEntrySize() {
```

Comment:
Since the entry size in memory/skip list will never have any padding I wanted to emphasize that this is the size of the entry when it is persisted. I am open to suggestions about the naming.